packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx

```diff
@@ -45,18 +45,7 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
       switchChecked={requirePayment}
       switchOnClick={(enabled) => {
         setRequirePayment(enabled);
-      }}
-      description={
-        <>
-          <div className="">
-            {t("payment_app_commission", {
-              paymentFeePercentage: 0.5,
-              fee: 0.1,
-              formatParams: { fee: { currency } },
-            })}
-          </div>
-        </>
-      }>
+      }}>
```

Comment:
I tried to find a link on stripe connect fees, but they are specific for each country.